pkg/network/tracer/ebpf_conntracker.go

```diff
@@ -57,6 +58,13 @@ type ebpfConntracker struct {
 
 // NewEBPFConntracker creates a netlink.Conntracker that monitor conntrack NAT entries via eBPF
 func NewEBPFConntracker(cfg *config.Config) (netlink.Conntracker, error) {
+	// dial the netlink layer aim to load nf_conntrack_netlink and nf_conntrack kernel modules
+	// eBPF conntrack require nf_conntrack symbols
+	conn, err := libnetlink.Dial(unix.NETLINK_NETFILTER, nil)
```

Comment:
If this fails, do we need to return an error?

Reply:
We will not be able to load ebpf or netlink conntrack so we will go the same error as today. The corner scenario would be if you upgrade your kernel but didn't reboot (the modules can't be loaded anymore)